build-system/pr-validation.yaml

```diff
@@ -78,7 +78,7 @@ jobs:
       outputDirectory: 'TestResults'
       artifactName: 'net_core_tests_linux-$(Build.BuildId)'
 
-  - template: azure-pipeline.template.yaml
+  - template: azure-pipeline.mntr-template.yaml
```

Comment:
Have to use different template here to add build artifacts that are not used for non-mntr tests. There is some conditional execution support, but it does not seem to be flexible enough

Reply:
Makes sense